go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/go-logr/logr v1.2.3
 	github.com/golang/mock v1.6.0
 	github.com/hashicorp/go-multierror v1.1.1
+	github.com/onsi/ginkgo v1.16.5
```

Comment:
I'm not too sure why this change is needed. I thought all of our unit/controller tests have moved to Ginkgo V2, and the E2E tests are in their own module.

Reply:
We want to be very conservative about introduction of new packages into CAPC itself.  These have potential to introduce a release-blocking security issue.

Reply:
++